packages/cli/src/commands/init.ts

```diff
@@ -923,14 +971,14 @@ async function initSubgraphFromExample(
     return;
   }
 
-  if (!isSubstreams) {
-    // Run code-generation
-    const codegen = await runCodegen(directory, commands.codegen);
-    if (codegen !== true) {
-      this.exit(1);
-      return;
-    }
-  }
+  // if (!isSubstreams) {
```

Comment:
Did we intentionally remove the `codegen` step here?

Reply:
Maybe that was removed as we are no longer asking for the `protocol`, but I think we can run `codegen` in all cases as it's a no-op for Substreams subgraphs?

Reply:
https://github.com/graphprotocol/graph-tooling/pull/1382/commits/4f03cf86d4d5b35fe775c1940189eaef3b0c9732